service/resource/deployment/resource_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	extensionsv1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
 )
 
+const ExpectedNodeCtrlCount = 1
```

Comment:
Please make this part of the test case. There are no package wide constants for the master and worker counts. So lets keep it within the test spec.

Reply:
ok